modules/c++/mem/unittests/test_scratch_memory.cpp

```diff
@@ -0,0 +1,117 @@
+/* =========================================================================
+ * This file is part of CODA-OSS
+ * =========================================================================
+ *
+ * (C) Copyright 2004 - 2018, MDA Information Systems LLC
+ *
+ * CODA-OSS is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include <mem/ScratchMemory.h>
+
+#include <mem/BufferView.h>
+#include <sys/Conf.h>
+#include "TestCase.h"
+
+namespace {
```

Comment:
Move the curly bracket down to the next line.

Reply:
Done.